build_tools/cmake/iree_cc_test.cmake

```diff
@@ -35,16 +35,6 @@ include(iree_installed_test)
 # This will also add it to ctest list as iree_${NAME}.
 #
 # Usage:
-# iree_cc_library(
-#   NAME
-#     awesome
```

Comment:
this is used by the sample test (`iree::awesome`)

Reply:
Oh yeah, that is what code reviews are for.... Sorry for the noise.